tests/utils.go

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package tests
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"math/big"
+	"strconv"
+	"strings"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/awm-relayer/messages/teleporter"
+	"github.com/ava-labs/subnet-evm/core/types"
+	"github.com/ava-labs/subnet-evm/ethclient"
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ava-labs/subnet-evm/tests/utils"
+	"github.com/ava-labs/subnet-evm/tests/utils/runner"
+	"github.com/ethereum/go-ethereum/common"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	defaultTeleporterTransactionGas       uint64 = 200_000
+	defaultTeleporterTransactionGasFeeCap        = big.NewInt(225 * params.GWei)
+	defaultTeleporterTransactionGasTipCap        = big.NewInt(params.GWei)
+	defaultTeleporterTransactionValue            = common.Big0
+)
+
+type SendCrossChainMessageEvent struct {
+	DestinationChainID ids.ID
+	MessageID          *big.Int
+	Message            teleporter.TeleporterMessage
+}
+
+// Teleporter contract sendCrossChainMessage input type
+type TeleporterMessageInput struct {
```

Comment:
Any reason to not name this `SendCrossChainMessageInput`?

Reply:
Renamed.